src/aead/aes.rs

```diff
@@ -292,6 +298,12 @@ pub enum Implementation {
     Fallback = 4,
 }
 
+#[cfg(target_arch = "mips64")]
+fn detect_implementation(_cpu_features: cpu::Features) -> Implementation {
```

Comment:
I believe the new version of this function in `master` will do the right thing automatically for you.